.eslintrc.js

```diff
@@ -0,0 +1,123 @@
+// turn these back on progressively
+const todo = {
+  'react/no-unused-prop-types': [
+    // turn to error after this cleanup story is done
+    'warn'
+  ],
```

Comment:
change this back to error after testing is complete
this is on warning because I've put `PropType` validation to fail for old removed/renamed prop-types, so that they get highlighted if they do happen to get used.